translate/api/Google.Cloud.Translate.V3.Samples/TranslateV3BatchTranslateText.cs

```diff
@@ -0,0 +1,108 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Generated code. DO NOT EDIT!
+
+// This is a generated sample ("LongRunningRequestPollUntilComplete", "translate_v3_batch_translate_text")
+
+using CommandLine;
+// [START translate_v3_batch_translate_text]
+using Google.Cloud.Translate.V3;
+using System;
+using System.Collections.Generic;
+
+namespace Google.Cloud.Translate.V3.Samples
+{
+    public class TranslateV3BatchTranslateText
+    {
+        /// <summary>
+        /// Batch translate text
+        /// </summary>
+        public static void SampleBatchTranslateText(string inputUri, string outputUri, string projectId, string location, string sourceLanguage, string targetLanguage)
+        {
+            TranslationServiceClient translationServiceClient = TranslationServiceClient.Create();
+            // TODO(developer): Uncomment and set the following variables
+            // string inputUri = "gs://cloud-samples-data/text.txt"
+            // string outputUri = "gs://YOUR_BUCKET_ID/path_to_store_results/"
+            // string projectId = "[Google Cloud Project ID]"
+            // string location = "us-central1"
+            // string sourceLanguage = "en"
+            // string targetLanguage = "ja"
+            BatchTranslateTextRequest request = new BatchTranslateTextRequest
+            {
+                ParentAsLocationName = new LocationName(projectId, location),
+                SourceLanguageCode = sourceLanguage,
+                TargetLanguageCodes =
+                {
+                    targetLanguage,
+                },
+                InputConfigs =
+                {
+                    new InputConfig
+                    {
+                        GcsSource = new GcsSource
+                        {
+                            InputUri = inputUri,
+                        },
+                        MimeType = "text/plain",
+                    },
+                },
+                OutputConfig = new OutputConfig
+                {
+                    GcsDestination = new GcsDestination
+                    {
+                        OutputUriPrefix = outputUri,
+                    },
+                },
+            };
+            // Poll until the returned long-running operation is complete
+            BatchTranslateResponse response = translationServiceClient.BatchTranslateText(request).PollUntilCompleted().Result;
+            Console.WriteLine($"Total Characters: {response.TotalCharacters}");
+            Console.WriteLine($"Translated Characters: {response.TranslatedCharacters}");
+        }
+    }
+
+    // [END translate_v3_batch_translate_text]
+
+    public class TranslateV3BatchTranslateTextMain
```

Comment:
As far as I can see, everything from here onward is just there for testing purposes. Please move to the test project.

Reply:
Are you suggesting to create C# file for each snippets in tests folder?

Reply:
If you could include it on the same file as the test itself that'd be great, although I understand this code being generated might complicate things. So yes, better have to files for each snippet on the test folder than all of this code here.
Remember this is a sample repo, some users, that might not be familiar with the client libraries, will get to this repo, download the code and play around with the samples. We don't want code that is meant to help with testing to be on the way for them, specially if they are not familiar with the libraries, it might take them a while to figure out what all of that code is for.